problems/5.loops.js

```diff
@@ -2,17 +2,29 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
+const evenNumbers = (num) => {
+  let newArray = 0
+  for (let i = 0; i <= number; i++)
```

Comment:
Missing {} on for loop. 
number is not defined. You need to call it num like you did in your parameter.